src/components/PageHeadingFlowRun.vue

```diff
@@ -85,12 +83,15 @@
     { text: flowRun.value?.name ?? '' },
   ])
 
-  const flowRunId = computed(() => props.flowRunId)
-  const flowRunSubscription = useSubscription(api.flowRuns.getFlowRun, [flowRunId], { interval: 30000 })
-  const flowRun = computed(() => flowRunSubscription.response)
+  const { flowRun } = useFlowRun(() => props.flowRunId, { interval: 30000 })
 
   const isPending = computed(() => flowRun.value?.stateType ? isPendingStateType(flowRun.value.stateType) : true)
-  const { taskRunsCount } = useTaskRunsCount(flowRunId)
+
+  const { count: taskRunsCount } = useTaskRunsCount(() => ({
+    flowRuns: {
+      id: [props.flowRunId],
+    },
+  }))
```

Comment:
The anonymous function makes this a little tougher to read than I'd hope (the one on line 86 is fine)

Reply:
Personally for small filters like this I prefer this and find it reads nicely. But if it gets much larger than this, yeah I'd wanna break it out.

Reply:
Ack - fine by me